content/english/articles/effective-gnops/onboarding-testnet-validator/index.md

```diff
@@ -1,11 +1,85 @@
 ---
 title: Onboarding a testnet validator
```

Comment:
This should probably be named `Road to Validation: How to become a testnet validator`.

And we should move this to the howto guides, it's not an `effective-gnops` article 🙏

Reply:
done in https://github.com/gnoverse/gnops/pull/27/commits/6f4e1f77b426dc6f69c79fd9afd974e064d944f4 and https://github.com/gnoverse/gnops/pull/27/commits/60cc93e77ccb323ed95dc629fccec9cf3b7e95b7